C/common/include/config_category.h

```diff
@@ -143,7 +145,7 @@ class ConfigCategory {
 					     const std::vector<std::string> options);
 				CategoryItem(const CategoryItem& rhs);
 				// Return both "value" and "default" items
-				std::string	toJSON(const bool full=false) const;
+				std::string	toJSON(const bool full=true) const;
```

Comment:
Why the optional parameter has a new default value? 

I do not recall exactly but some routines are calling  toJSON() and others toJSON(true)

Reply:
It is possibly not needed. The problem is caused by incomplete configuration categories being sent to the configuration manager for merge, hence always sending everything. Will experiment further to see if this is needed.